setupFetchForJest.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright 2023 Inclusive Design Research Centre, OCAD University
+ * All rights reserved.
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/inclusive-design/adaptive-palette/blob/main/LICENSE
+ */
+
+import { fetch } from "whatwg-fetch";
```

Comment:
If you happen to have the solution article about using this npm package to fix the use of `fetch()` in the Jest, can you add a code comment to explain what this script is for as well as including the article? Thanks.

Reply:
Sure.  I have added a comment/link.